ec2/multi_node/README.md

```diff
@@ -0,0 +1,131 @@
+# Multi-Node AWS Openshift Deployment 
+OpenShift environment with the Service Catalog & Ansible Service Broker configured in AWS with multiple EC-2 instances
+
+## Overview
+These playbooks will:
+  * Create a public VPC if it does not exist
+  * Create a security group if it does not exist
+  * Create a configurable number of instances with a specific Name if does not exist
+  * Configure a hostname with as a CNAME record through Route53
+  * Setup OpenShift via the [Openshift Ansible BYO playbook](https://github.com/openshift/openshift-ansible/blob/master/playbooks/byo/config.yml)
+  * Install Service Catalog 
+  * Install Ansible Service Broker
+  * Configure cluster users with htpasswd authentication
+
+## Pre-Reqs
+  * Ansible needs to be installed so its source code is available to Python.
+    * Check to see if Ansible modules are available to Python
+      ```bash
+      $ python -c "import ansible;print(ansible.__version__)"
+      2.3.0.0
+      ```
+    * MacOS requires Ansible to be installed from `pip` and not `brew`
+      ```bash
+      $ python -c "import ansible;print(ansible.__version__)"
+      Traceback (most recent call last):
+      File "<string>", line 1, in <module>
+      ImportError: No module named ansible
+
+      brew uninstall ansible
+      pip install ansible
+
+      $ python -c "import ansible;print(ansible.__version__)"
+      2.3.0.0
+      ```
+  * Install python dependencies (This is needed for python2. Use pip2 if using python3)
+    * On Fedora and EL7 it is recommended that you use ansible in a python virtualenv.
+     * This is due to a couple reasons:
+       - boto rpms are not sufficiently new enough
+       - pip is not sudo safe on Fedora and EL7 
+     * To setup and active a virtualenv do the following;
```

Comment:
activate. Sorry. I know I wrote this... I apparently did not have enough coffee when I did it...